backend/index.js

```diff
@@ -0,0 +1,15 @@
+const { Router } = require('express');
```

Comment:
Let's move this file inside a directory named `src`. This way, the `backend/src` directory will contain all of the code.

Reply:
Done sir